app/models/user.rb

```diff
@@ -14,4 +13,8 @@ class User < ApplicationRecord
   has_secure_password
 
   enum role: ['user', 'merchant', 'admin']
+
+  def disable_merchant_items
```

Comment:
To put more work on the database and avoid iterating over all the items one at a time, could we use a `.update_all` call here?  Here's the [documentation](https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/update_all) on it, it sounds like we will need to manually change the update_at timestamp though.

This would also dry up our code by removing the Item.disable method.  Although it might be needed on a merchant user story if they want to just disable one item so maybe leave it.

Reply:
Agreed to keep the current methods to change items until further review